Makefile

```diff
@@ -160,14 +160,19 @@ olm-verify:
 	hack/dockerized "./hack/olm.sh verify"
 
 current-dir := $(realpath .)
+rule-spec-dumper-executable := "rule-spec-dumper"
 
 build-prom-spec-dumper:
-	hack/dockerized "go build -o rule-spec-dumper ./hack/prom-rule-ci/rule-spec-dumper.go"
+	hack/dockerized "go build -o ${rule-spec-dumper-executable} ./hack/prom-rule-ci/rule-spec-dumper.go"
+
+clean-prom-spec-dumper:
+	rm -f ${rule-spec-dumper-executable}
 
 prom-rules-verify: build-prom-spec-dumper
 	./hack/prom-rule-ci/verify-rules.sh \
-		"${current-dir}/rule-spec-dumper" \
+		"${current-dir}/${rule-spec-dumper-executable}" \
 		"${current-dir}/hack/prom-rule-ci/prom-rules-tests.yaml"
+	$(MAKE) clean-prom-spec-dumper
```

Comment:
Being curious: why is the rebuild required every time?

Reply:
Thanks Daniel. I've removed this.